LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -291,6 +311,18 @@ function removeMember(groupID, userID) {
     });
 }
 
+function addNexusMember(groupID, empUID) {
```

Comment:
Need a fail catch.  Likely just a log output with the reason so we can track issues.